[2025-02-14T10:33:51.292+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T10:33:51.307+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: create_file_dag.create_file_task manual__2025-02-14T10:33:49.673819+00:00 [queued]>
[2025-02-14T10:33:51.315+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: create_file_dag.create_file_task manual__2025-02-14T10:33:49.673819+00:00 [queued]>
[2025-02-14T10:33:51.316+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-14T10:33:51.454+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): create_file_task> on 2025-02-14 10:33:49.673819+00:00
[2025-02-14T10:33:51.464+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=260) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T10:33:51.465+0000] {standard_task_runner.py:72} INFO - Started process 262 to run task
[2025-02-14T10:33:51.465+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'create_file_dag', 'create_file_task', 'manual__2025-02-14T10:33:49.673819+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/create_file_dag.py', '--cfg-path', '/tmp/tmpqc___cih']
[2025-02-14T10:33:51.466+0000] {standard_task_runner.py:105} INFO - Job 41: Subtask create_file_task
[2025-02-14T10:33:51.500+0000] {task_command.py:467} INFO - Running <TaskInstance: create_file_dag.create_file_task manual__2025-02-14T10:33:49.673819+00:00 [running]> on host 7b78e6c4b0cc
[2025-02-14T10:33:51.553+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='create_file_dag' AIRFLOW_CTX_TASK_ID='create_file_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T10:33:49.673819+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T10:33:49.673819+00:00'
[2025-02-14T10:33:51.554+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-14T10:33:51.555+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-14T10:33:51.556+0000] {logging_mixin.py:190} INFO - Current task name:create_file_task state:running start_date:2025-02-14 10:33:51.308177+00:00
[2025-02-14T10:33:51.556+0000] {logging_mixin.py:190} INFO - Dag name:create_file_dag and current dag run status:running
[2025-02-14T10:33:51.556+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-14T10:33:51.557+0000] {logging_mixin.py:190} INFO - 檔案 /opt/***/1 已經創建！
[2025-02-14T10:33:51.557+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-14T10:33:51.565+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-14T10:33:51.565+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=create_file_dag, task_id=create_file_task, run_id=manual__2025-02-14T10:33:49.673819+00:00, execution_date=20250214T103349, start_date=20250214T103351, end_date=20250214T103351
[2025-02-14T10:33:51.581+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-14T10:33:51.581+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-14T10:33:51.582+0000] {logging_mixin.py:190} INFO - Dag name:create_file_dag queued_at:2025-02-14 10:33:49.682934+00:00
[2025-02-14T10:33:51.582+0000] {logging_mixin.py:190} INFO - Task hostname:7b78e6c4b0cc operator:PythonOperator
[2025-02-14T10:33:51.599+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T10:33:51.611+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-14T10:33:51.613+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
